governance/program/proptest-regressions/state/proposal.txt

```diff
@@ -0,0 +1,3 @@
+cc c4e184f7da9cb0fadb2b7e2064e1797262f89324176acd14b84cf40098aea268 # shrinks to state = Draft
```

Comment:
Do these need to be committed?  Typically it's for use cases that have failed and been fixed in the past.  Since the proptests you're using don't seem to have too many possible cases, these might have come from initial development

Reply:
Removed